classes/class-ppc-update.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Pre-Publish Checklist.
+ *
+ * PHP version 7
+ *
+ * @category PHP
+ * @package  Pre-Publish Checklist.
+ * @author   Display Name <username@brainstormforce.com>
+ * @license  http://brainstormforce.com
+ * @link     http://brainstormforce.com
+ */
+
+if ( ! class_exists( 'PPC_Update' ) ) :
```

Comment:
Remove class_exists check. This class already has a prefix. Also, this is defined from plugin itself. We are not using a third party plugin class.

Reply:
class_exists removed.